### ISSUE TITLE
Static IP not set on GKE

### ISSUE THREAD

**Issue by fargozhu** [SUSE/kubecf#291]
**Describe the bug**

The static IPs to configure the services: **router**, **ssh-proxy**, and **tcp-router,** are not being passed to GKE during the deployment phase and so we end up using the ones assigned by GKE instead.

**To Reproduce**

**values.yaml** file

```
system_domain: kubecf.awesome-domain.io

features:
  eirini:
    enabled: true
  ingress:
    enabled: false
    
services:
  router:
    externalIPs:
    - 35.214.153.155
  ssh-proxy:
    externalIPs:
    - 35.214.130.126
  tcp-router:
    externalIPs:
    - 35.214.205.136

kube:
  pod_cluster_ip_range: 10.24.0.0
  service_cluster_ip_range: 10.88.0.0
```

**Deploy KubeCF**

```
helm install kubecf --namespace kubecf --values values.yaml https://github.com/SUSE/kubecf/releases/download/v0.1.0/kubecf-0.1.0-002b49a.tgz
```

**Output**

```
kubecf        kubecf-router-public           LoadBalancer   10.97.11.40    35.204.237.80,35.214.153.155   80:31251/TCP,443:30482/TCP                                                                                                                        37m
kubecf        kubecf-routing-api             ClusterIP      None           <none>                         3000/TCP                                                                                                                                          26m
kubecf        kubecf-routing-api-0           ClusterIP      10.97.11.185   <none>                         3000/TCP                                                                                                                                          26m
kubecf        kubecf-singleton-blobstore     ClusterIP      None           <none>                         8080/TCP,4443/TCP                                                                                                                                 26m
kubecf        kubecf-singleton-blobstore-0   ClusterIP      10.97.5.212    <none>                         8080/TCP,4443/TCP                                                                                                                                 26m
kubecf        kubecf-ssh-proxy-public        LoadBalancer   10.97.10.7     35.204.62.177,35.214.130.126   2222:30238/TCP                                                                                                                                    37m
kubecf        kubecf-tcp-router              ClusterIP      None           <none>                         80/TCP,20000/TCP,20001/TCP,20002/TCP,20003/TCP,20004/TCP,20005/TCP,20006/TCP,20007/TCP,20008/TCP                                                  26m
kubecf        kubecf-tcp-router-0            ClusterIP      10.97.6.13     <none>                         80/TCP,20000/TCP,20001/TCP,20002/TCP,20003/TCP,20004/TCP,20005/TCP,20006/TCP,20007/TCP,20008/TCP                                                  26m
kubecf        kubecf-tcp-router-public       LoadBalancer   10.97.12.74    34.90.124.195,35.214.205.136   20000:32260/TCP,20001:30608/TCP,20002:32284/TCP,20003:30674/TCP,20004:31939/TCP,20005:31723/TCP,20006:31991/TCP,20007:30245/TCP,20008:30170/TCP   37m
kubecf        kubecf-uaa                     ClusterIP      None           <none>                         8080/TCP,8443/TCP                                                                                                                                 26m
kubecf        kubecf-uaa-0                   ClusterIP      10.97.8.103    <none>                         8080/TCP,8443/TCP                                                                                                                                 26m
```

where the 2nd IP associated with each LoadBalancer service is only a declaration set during the deployment phase.

**Expected behavior**
The IPs from the values.yaml must be passed to the GKE to be used instead of the ones randomly assigned by the cloud provider.

**Environment**
 - KubeCF: 0.1.0-002b49a
 - cf-operator: v1.0.0-1.g424dd0b3

**Additional context**

**Comment by drnic** [comment]
GKE docs suggest a single static IP can be requested for a LoadBalancer service under `spec.loadBalancerIP`, rather than an array of IPs at `spec.externalIPs`.

https://cloud.google.com/kubernetes-engine/docs/tutorials/configuring-domain-name-static-ip

**Comment by f0rmiga** [comment]
@prabalsharma Could you check on EKS with the fix from #296, please?
